support-frontend/app/controllers/Application.scala

```diff
@@ -141,9 +141,25 @@ class Application(
     ).map(_.withSettingsSurrogateKey)
   }
 
+  private def shareImageUrl(geoData: GeoData): String = {
+    val ausMomentEnabled =
+      settingsProvider
```

Comment:
it's preferable here for the calling function to pass in `settings` rather than calling `getAllSettings()` again here.
So: `private def shareImageUrl(geoData: GeoData, settings: AllSettings): String`